src/Analyzers/MSTest.Analyzers/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,114 @@
+﻿#nullable enable
```

Comment:
Not sure why but the analyzer suddenly triggered issues on the project although the configuration to not raise when the file doesn't exist is setup.